### PR TITLE
Optional display of pylint code, and human-readable version.

### PR DESCRIPTION
**Pull request by bmhkim**
Added display of pylint code, and human-readable (short) code, depending on new setting options. This solves issue #28.

### CODE REVIEW

File: lib/init.coffee [hunk 1]
```diff
@@ -6,6 +6,14 @@ module.exports =
     rcFile:
       type: 'string'
       default: ''
+    showCodes:
+      description: "Show pylint codes? (E1101, etc.)"
```
Comment (posita) [inline lib/init.coffee]: Consider instead something like:

title: "Show Pylint Codes"
description: "Displays the Pylint code in the Linter message  (e.g., E1101, etc.)"

File: lib/init.coffee [hunk 2]
```diff
@@ -6,6 +6,14 @@ module.exports =
...
+      type: 'boolean'
+      default: false
+    showReadableCodes:
+      description: "Show the short human-readable version of pylint codes? (e.g., unused-import for W0611)"
```
Comment (posita) [inline lib/init.coffee]: Consider instead something like:

title: "Show Pylint Symbols"
description: "Displays the human-readable Pylint symbol in the Linter message  (e.g., unused-import, etc.)"

**Comment by posita** [comment]
Not my rodeo, but this gets a :+1: from me. Consider my above suggestions regarding the separating the title and descriptions for the settings. Also consider [squashing your second (and any subsequent) commit](http://gitready.com/advanced/2009/02/10/squashing-commits-with-rebase.html) to make this PR more attractive to the maintainer.

**Comment by bmhkim** [comment]
Thanks for the comments, I appreciate the feedback. Since I agree with all you've said, I've updated the option descriptions to be declarative statements describing the behavior being controlled, and squashed my commits. I imagine you'll need to ignore some git warnings....

**Comment by posita** [comment]
LGTM

@dmnd, thoughts?

**Comment by SpainTrain** [comment]
Should this be closed now that #47 is merged?

**Comment by posita** [comment]
@SpainTrain, is it validated as working and released?

**Comment by SpainTrain** [comment]
Definitely not released.

I tried to validate it works, but I think an unrelated change that's currently on master made the plugin not work at all (at least in my environment).

Agree it makes sense to keep it open until some folks can verify that #47 actually works.

**Comment by SpainTrain** [comment]
> @SpainTrain, is it validated as working and released?

Approach in #47 works for me.  Same approach works in #55. One validation data point X-D

**Comment by SpainTrain** [comment]
Definitely appreciate the work, but the code has changed significantly since this was opened and similar functionality was implemented. Closing.

**Pull request closed without merging**